pkg/parser/local_orb.go

```diff
@@ -63,18 +63,54 @@ func (doc *YamlDocument) parseLocalOrb(name string, orbNode *sitter.Node) (*Loca
 		return nil, err
 	}
 
+	orbInfo := &ast.OrbInfo{
+		IsLocal: true,
+
+		Source:      deindentedContent,
+		Description: orbDoc.Description,
+		Commands:    orbDoc.Commands,
+		Jobs:        orbDoc.Jobs,
+		Executors:   orbDoc.Executors,
+		Parameters:  orbDoc.PipelinesParameters,
+
+		OrbsRange:       utils.AddOffsetToRange(orbDoc.OrbsRange, orb.Offset),
+		ExecutorsRange:  utils.AddOffsetToRange(orbDoc.ExecutorsRange, orb.Offset),
+		CommandsRange:   utils.AddOffsetToRange(orbDoc.CommandsRange, orb.Offset),
+		JobsRange:       utils.AddOffsetToRange(orbDoc.JobsRange, orb.Offset),
+		WorkflowRange:   utils.AddOffsetToRange(orbDoc.WorkflowRange, orb.Offset),
+		ParametersRange: utils.AddOffsetToRange(orbDoc.PipelinesParametersRange, orb.Offset),
+	}
+
+	doc.LocalOrbInfo[name] = orbInfo
+
+	// Commands
 	for name, command := range orbDoc.Commands {
-		doc.Commands[fmt.Sprintf("%s/%s", orb.Name, name)] = doc.adaptCommand(orb, command)
+		commandName := fmt.Sprintf("%s/%s", orb.Name, name)
+		orbCommand := doc.adaptCommand(orb, command)
+
+		orbDoc.Commands[name] = orbCommand
+		doc.Commands[commandName] = orbCommand
```

Comment:
I don't understand why we still do this for commands, jobs and executors, we shouldn't IMO

Reply:
Because commands defined in the orbs can be use in the orb itself and from outside the orb.